src/helper/bitmap.js

```diff
@@ -275,8 +275,23 @@ const drawEllipse = function (options, context) {
     if (!matrix.isInvertible()) return false;
     const inverse = matrix.clone().invert();
 
+    const isGradient = context.fillStyle instanceof CanvasGradient;
+
+    // If drawing a gradient, we need to draw the shape onto a temporary canvas, then draw the gradient atop that canvas
+    // only where the shape appears. Some points get double-filled which is a problem if the gradient fades to
```

Comment:
Clarify that this is an issue with the drawShearedEllipse function that could be a problem if we become able to draw semitransparent ellipses. Maybe link to an issue?